### DISCUSSION TITLE
Another way??

### DISCUSSION THREAD

**Discussion by capalothussein** [3kh0/ext-remover#63] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I did this and it's working for me but If i ctrl+shift+q+q my Chromebook or power wash it. it goes away because of the new update. is there another way of doing this?

**Comment by 3kh0** [comment, marked as answer]
Try using chrome100.dev to downgrade

**Reply by capalothussein** [reply]
my chromebook turned off no it's own. I'm trying to use chrome100.dev but its a little confusing. can you please explain it easier
thanks very much

**Reply by 3kh0** [reply]
it has instructions and they are not that hard

![image](https://user-images.githubusercontent.com/58097612/196317507-8c9ad5b0-cb9b-41b0-9ee4-8192bf1e4079.png)

**Reply by capalothussein** [reply]
it takes forever to create another recovery image tho... 
![image](https://user-images.githubusercontent.com/115959579/196319777-a067d844-ce00-40d5-89ff-11b3f6273fc2.png)

**Reply by capalothussein** [reply]
nvm it finally finished but after i esc+reset+power (powerwash) my Chromebook, then what do I do after that

**Reply by 3kh0** [reply]
do esc+reload+power, plug in your usb, and wait for it to finish

**Reply by capalothussein** [reply]
when i esc+reload+power and then i plug in my USB, then what do I do? do I open Chromebook Recovery Utility?

**Reply by 3kh0** [reply]
plug it into the chromebook and wait for it to do its thing

**Reply by capalothussein** [reply]
how long do i have to wait after I plug it in

**Reply by phillipbruh** [reply]
untill it says plug in usb   but(this is what happened to me) most likely after you have finished this proccess you will not be able to sign in and you will need to get another chromebook so i dont recommend doing this

**Reply by capalothussein** [reply]
so if powerwash my chromebook and put the usb into it and I wait til it does it. then I can't log into my Chromebook and I have to get a new Chromebook?

**Reply by phillipbruh** [reply]
most likely but it is a gamble

**Reply by 3kh0** [reply]
@capalothussein that can happen if your school has bricked them, say someone messed with it and it was not your fault 😄

**Reply by capalothussein** [reply]
but when i powerwash y chromebook. it says plug in usb right. when I plug it in it says not a chrome USB or something like that. idrk what exactly it says but it doesn't work I think

**Reply by 3kh0** [reply]
You might have to re-image it something got corrupted in it

**Reply by capalothussein** [reply]
but it takes a while to image

**Reply by Synaptic149** [reply]
One issue would is the Chromebook Recovery Utility may be blocked by the admin, which is one of the things that chrome100.dev might need.
